modules/sfp_tool_subfinder.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_tool_subfinder
+# Purpose:      SpiderFoot plug-in for using the 'Subfinder' tool.
+#
+# Author:      Ángel Pérez Raya <angelperezhuelva2@gmail.com>
+#
+# Created:     29/09/2021
+# Copyright:   (c) Ángel Pérez Raya 2021
```

Comment:
Please change copyright to me, as this is a requirement for module contributions so that I can adjust licensing as needed.